### PR TITLE
port to ROS2

### PR DESCRIPTION
**Pull request by SyZbidi**
Ported package to ROS2

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,64 +1,22 @@
...
-generate_messages(
-  DEPENDENCIES
-    actionlib_msgs
-    geometry_msgs
+rosidl_generate_interfaces(${PROJECT_NAME}
+  "action/FollowCartesianTrajectory.action"
+  "msg/CartesianPosture".msg"
+  "msg/CartesianTolerance".msg"
+  "msg/CartesianTrajectory".msg"
+  "msg/CartesianTrajectoryPoint".msg"
```
Comment (gavanderhoorn) [inline CMakeLists.txt:16]:
```suggestion
  "msg/CartesianPosture.msg"
  "msg/CartesianTolerance.msg"
  "msg/CartesianTrajectory.msg"
  "msg/CartesianTrajectoryPoint.msg"
```

**Comment by gavanderhoorn** [comment]
@fmauch: would you be interested in merging this here? It would save us from maintaining an(other) fork and might help others trying to use this in ROS 2 as well.

There's no CI currently here, correct?

**Comment by fmauch** [comment]
Yes, that would be nice to have here. Not necessarily the main branch, though.

**Comment by SyZbidi** [comment]
> Yes, that would be nice to have here. Not necessarily the main branch, though.

If you can create a new ROS2-dedicated branch to which I can target this PR that would be great. Thanks

**Review by fmauch — APPROVED** [review]
This is looking good at least for an initial version. I'll create a ROS 2 branch, merge this PR manually, update CI for ROS 2 and push that to Github. Thanks @SyZbidi

**Comment by gavanderhoorn** [comment]
There are still one or two changes missing @fmauch. @SyZbidi was going to push those in a bit.

**Comment by fmauch** [comment]
I'll take some time to do that anyway, since I'll have to squeeze that in during childcare. I wouldn't push things before I've tested them manually, anyway.

**Comment by gavanderhoorn** [comment]
We have them locally. Just finishing lunch :)

**Comment by SyZbidi** [comment]
@fmauch Done with the final changes, happy to contribute :)

**Comment by SyZbidi** [comment]
> @SyZbidi could you rebase on `ros2-main` please? that should make the workflows run correctly.

Done

**Comment by fmauch** [comment]
Thank you @SyZbidi!

**Pull request merged**